libraries/TGRSIAnalysis/TAngularCorrelation/TAngularCorrelation.cxx

```diff
@@ -418,7 +547,8 @@ TGraphAsymmErrors* TAngularCorrelation::CreateGraphFromHst(TH1* hst)
 
       // fill graph with point
       Int_t graphn = graph->GetN();
-      graph->SetPoint(graphn,TMath::Cos(angle),y);
+      if(!fold) graph->SetPoint(graphn,TMath::Cos(angle),y);
+      else if(fold) graph->SetPoint(graphn,angle,y);
```

Comment:
I found the answer later on - I suggest we actually save the angle values in radians and eliminate this if statement.